.rubocop.yml

```diff
@@ -81,3 +81,6 @@ AllCops:
     - 'lib/templates/chat/routes_2.rb'
     - 'lib/templates/chat/sessions_controller.rb'
     - 'lib/templates/chat/specs/rails_helper.rb'
+    - 'lib/templates/facebook/sessions_routing_spec.rb'
+    - 'lib/templates/facebook/sessions_controller.rb'
+    - 'lib/templates/facebook/factories_user.rb'
```

Comment:
Why do you have to exclude from AllCops?

Reply:
This was because there was no way to exclude the rule "No new line at end of file", but I'll fix it after talking with @grilix  and remove this exceptions